server/src/main/java/org/elasticsearch/index/query/functionscore/LinearDecayFunctionBuilder.java

```diff
@@ -65,6 +65,7 @@ private static final class LinearDecayScoreFunction implements DecayFunction {
 
         @Override
         public double evaluate(double value, double scale) {
+            // throw new IllegalArgumentException("HELLO: [" + value + "] [" + scale + "]");
```

Comment:
Should we remove this line ;) ?

Reply:
Haha, yes! Thanks for catching this :facepalm: .